kernel/kernel-impl/src/test/java/org/sakaiproject/util/impl/FormattedTextTest.java

```diff
@@ -1052,4 +1052,27 @@ public void testGetShortenedTitles() {
         }
     }
 
+    @Test
```

Comment:
I know it's probably not completely relevant to the patch but what does jsoup do/return if the tags are unbalanced or not really tags? Like for instance 

"<table>this is a table" //I'd don't know what I'd expect

"a<b>d" // I know this is a hard one but I'd probably expect a<b>d back?

Just interesting additional test cases I guess?